File: src/main/java/me/centralhardware/znatoki/telegram/country/days/tracker/Dto/Track.java
package me.centralhardware.znatoki.telegram.country.days.tracker.Dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Builder
@Getter
@Setter
public class Track {

    private LocalDateTime dateTime;
    private Long userId;
    private Float latitude;
    private Float longitude;
    private Integer altitude;
    private String country;

}

File: src/main/kotlin/Dto/Track.kt
package me.centralhardware.znatoki.telegram.country.days.tracker.Dto

import java.time.LocalDateTime

data class Track(
    val dateTime: LocalDateTime,
    val userId: Long,
    val latitude: Float,
    val longitude: Float,
    val altitude: Int,
    val country: String
)
